python/cudf/cudf/core/dataframe.py

```diff
@@ -5860,9 +5861,11 @@ def _reduce(
             )
             source = self._get_columns_by_label(numeric_cols)
             if source.empty:
-                return Series(index=self.index)
-
-        axis = source._get_axis_from_axis_arg(axis)
+                return Series(
+                    index=self._data.to_pandas_index()[:0]
```

Comment:
The `[:0]` is to get the empty `Index` with the correct type?

Reply:
Yup